docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,819 @@
+::
+
+## Overview
+
+Sub-path routing in Alokai uses the [Config Switcher extension](https://docs.alokai.com/middleware/guides/config-switcher) with the header strategy. This allows your Middleware to dynamically select the appropriate configuration based on which store the user is browsing.
+
+For example, with this setup you could have:
+- `/en/electronics` - Electronics store with its specific configuration
+- `/en/apparel` - Apparel store with different products, styling, and configuration
+
+All of this is served from a single Frontend deployment.
+
+### How Config Switcher Works
+
+The Config Switcher dynamically changes your middleware integration configuration based on the current request configuration ID sent with the `x-alokai-middleware-config-id` header. This means:
+
+1. **Path-Based Configuration**
+    - Different configuration ID set by different paths allow to fetch data from different sources
+    - Example: `/apparel` path shows apparel products, while `/electronics` shows electronics products
+
+2. **Configuration ID vs Store ID**
+    - Each configuration has its own distinct ID
+    - This is different from the store ID (`storeId`)
+    - Multiple configurations can exist within a single store
+
+3. **Same-Store Operation**
+    - Config Switcher works within a single store instance
+    - It extracts the configuration ID from the request
+    - It executes requests using the appropriate configuration
+
+::tip Config Switcher Documentation
+This section provides a brief overview of setting up Config Switcher. For complete documentation, refer to the [Config Switcher guide](https://docs.alokai.com/middleware/guides/config-switcher).
+::
+
+### Key Distinctions To File-Based Inheritance
+
+`ConfigSwitcher` operates differently from file-based inheritance:
+
+| **File-Based Inheritance**              | **Config Switcher**                                              |
+| --------------------------------------- | ---------------------------------------------------------------- |
+| Creates separate deployable instances   | Uses a single deployable instance                                |
+| Provides inheritance mechanism to make managing multiple stores easy             | You have to manage codebase reuse between stores on your own |
+| Requires separate builds for each store | Single build supports multiple configurations                    |
+
+Take a look at the diagram to see the difference:
+
+<img src="../../img/file-based-vs-switcher-diagram.png" alt="Comparison of File-based Inheritance vs Config Switcher" class="mx-auto">
+
+## Setting Up the Middleware
+
+The first step is to configure your Middleware to support different configurations using the Config Switcher extension.
+
+Create a Config Switcher extension for your integration:
+
+```ts
```

Comment:
I think this would be the correct syntax for adding file path to a fenced code block
```suggestion
```ts{}[apps/storefront-middleware/integrations/<integration-name>/extensions/configSwitcher.ts]
```

Reply:
I was using this syntax in the past, but other team members in multibrand team said that it's better to set the file path as a comment, because the UI element for the file path looks bad when the path is long. We didn't have capacity that time to improve this UI element, so I've decided to use the comment instead